plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -277,13 +281,16 @@ export class EventsProcessor {
         snapshot_data: Record<any, any>,
         properties: Properties,
         ip: string | null
-    ): Promise<PreIngestionEvent> {
+    ): Promise<IngestionEvent> {
+        if (!timestamp) {
```

Comment:
how would we get null timestamps here?

Reply:
Hm, I think this was leftover from a merge with `master, removed